ome/figure
# File open dialog

This improves the File > Open dialog in a couple of ways.
Faster loading of files from OMERO (using projection query and not parsing json).
We now parse the json from file description during the Javascript render() of files.
The handling of missing thumbnails for deleted images is now done by the webgateway render_thumbnail (same as for webclient) instead of doing an extra query to identify missing images.

To test:
 - Try opening files in figure. Check for any problems. Should work as before.
 - When files are being loaded (and on "Refresh"), you should see a "Loading Files..." placeholder.
 - Loading of files should be faster, although I've previously only noticed it being slow on nightshade (7 secs to load 135 files).
 - The currently opened file should be "disabled" (not clickable) in the File > Open list. Should also be "disabled" after hitting refresh.
 - Try opening another user's File (where canEdit is False) and check that this is still reflected in the UI - shouldn't be able to Save change.
